.env

```diff
@@ -9,7 +9,8 @@ NEXT_PUBLIC_ALGOLIA_SEARCH_ONLY_API_KEY=2cd9898a6c253bfa3965d2b62a4f7f3d
 # which takes precedence over the NEXT_PUBLIC_ALGOLIA_INDEX env var.
 # NEXT_PUBLIC_ALGOLIA_INDEX=product_WAYPOINT
 
-MKTG_CONTENT_API="https://content.hashicorp.com"
+# MKTG_CONTENT_API="https://content.hashicorp.com"
+MKTG_CONTENT_API="https://mktg-content-api-git-pre-release-api-stub-hashicorp.vercel.app"
```

Comment:
Be sure to undo this change

Reply:
Yeah this is just temporary to allow reviewers to test locally. I'll undo this once it looks good to go.